### ISSUE TITLE
Inconsistent syntax with original fzf

### ISSUE THREAD

**Issue by SimonMandlik** [nvim-telescope/telescope-fzf-native.nvim#52]
Combination of NOT (`!`) operator and OR (`|`) works differently than in vanilla fzf.

For example (on this repo with default telescope + fzf-native config):

<img width="686" alt="image" src="https://user-images.githubusercontent.com/16707112/148662923-abc571ff-78c8-4cda-91b4-bee4d19b6b71.png">

whereas command line fzf gives me

<img width="329" alt="image" src="https://user-images.githubusercontent.com/16707112/148662979-9f059cf2-4750-4d66-80d2-335da5582992.png">

**Comment by Conni2461** [comment]
I don't guarantee completeness: https://github.com/nvim-telescope/telescope-fzf-native.nvim#disclaimer because its not a 1:1 copy of the go code.

But this looks like a bug in my prompt parsing. I'll look into it if i find the time

**Comment by Conni2461** [comment]
finally found some time fixing this issue: https://github.com/nvim-telescope/telescope-fzf-native.nvim/pull/56

Could you try out this PR? dont forget to rerun make :) Thanks

Also please mention if you found any regressions, the tests dont indicate any but i am not sure right now. I havent looked at this code in a while

**Comment by Conni2461** [comment]
`!a | !b` is also borked

**Comment by SimonMandlik** [comment]
Yes, there are still some problems. `!a | !b` doesn't work for me either
